kong/plugins/acl/daos.lua

```diff
@@ -12,6 +12,7 @@ return {
       { created_at = typedefs.auto_timestamp_s },
       { consumer = { type = "foreign", reference = "consumers", required = true, on_delete = "cascade" }, },
       { group = { type = "string", required = true } },
+      { tags         = typedefs.tags },
```

Comment:
style: excess spacing in schema entries? (this and the other `daos.lua` in this PR)

Reply:
Thanks, fixed!